python/cli/prelude_cli/views/build.py

```diff
@@ -343,7 +343,7 @@ def update_threat_hunt(controller, threat_hunt, name, query, test):
             name=name,
             query=query,
             test_id=test,
-            threat_hunt_id=threat_hunt,
+            threat_hunt_id=threat_hunt
```

Comment:
super nitty nit: I personally prefer trailing `,` because it makes future PR diffs smaller, but we don't really have a standard for it

Reply:
it's just feels so unfinished ha but ok

Reply:
I don't have strong enough feelings to say we have to do it. I just like it when diffs are one line instead of two. We can standardize the other way if you have opinions

Reply:
i'll try to get in the habit of the trailing comma for multiline lists / params